src/internal/pjs/api_server.go

```diff
@@ -111,3 +112,92 @@ func (a *apiServer) InspectJob(ctx context.Context, req *pjs.InspectJobRequest)
 		},
 	}, nil
 }
+
+func (a *apiServer) ProcessQueue(srv pjs.API_ProcessQueueServer) (retErr error) {
+	ctx := srv.Context()
+	req, err := srv.Recv()
+	if err != nil {
+		return errors.EnsureStack(err)
```

Comment:
try not to use errors.EnsureStack() and favor errors.Wrap() instead. We really should mark EnsureStack as deprecated.

Reply:
Got it.